support-frontend/assets/pages/[countryGroupId]/components/paymentFields.ts

```diff
@@ -181,6 +182,8 @@ export const getPaymentFieldsForPaymentMethod = async (
 	if (paymentMethod === 'StripeHostedCheckout') {
 		return {
 			paymentType: StripeHostedCheckout,
+			checkoutSessionId: checkoutSessionId ?? '',
```

Comment:
I wonder if there's anything we can do to avoid having to pass the empty string here. I think we did this because there are two versions of the payment fields - one when we first submit the form to the server where we haven't got a checkout session ID yet - we're about to create it but it doesn't exist when the submit happens, and then another when the user has been through the Stripe flow and we have got a checkout session ID, which is used by support-workers to get the payment method.

Reply:
I'll have a look at refactoring to an option

Reply:
That was pretty easy!